cvat/apps/webhooks/serializers.py

```diff
@@ -93,6 +93,10 @@ class Meta:
             "last_delivery_date",
         )
         read_only_fields = fields
+        extra_kwargs = {
+            "project": {"allow_null": True},
```

Comment:
Why do we need "project"? I see `project_id` in the list above of available fields.

Reply:
Removed.